config/opal_setup_cli.m4

```diff
@@ -0,0 +1,38 @@
+dnl -*- shell-script -*-
+dnl
+dnl Copyright (c) 2016      Research Organization for Information Science
+dnl                         and Technology (RIST). All rights reserved.
+dnl $COPYRIGHT$
+dnl
+dnl Additional copyrights may follow
+dnl
+dnl $HEADER$
+dnl
+
+
+# Capture configure command line in the parameter and do the AC substitution
```

Comment:
Suggest either making this a "dnl" comment (so that it is not rendered in the output `configure` script), or moving it inside the AC_DEFUN (so that it is rendered alongside the code that does what it describes).  As it is now, this line will be rendered in `configure` at the moment of inclusion, vs. when `OPAL_CAPTURE_CONFIGURE_CLI` is invoked.  Hence, this comment line will likely be quite far away from the code rendered by `OPAL_CAPTURE_CONFIGURE_CLI`.

We're probably not careful about this in other .m4 files, but we might as well not compound the issue.  😄